angular/src/pipes/color-password-count.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { Pipe } from '@angular/core';
+import { ColorPasswordPipe } from './color-password.pipe';
+
+/*
+ An updated pipe that extends ColourPasswordPipe to include a character count
+*/
+@Pipe({ name: 'colorPasswordCount' })
+export class ColorPasswordCountPipe extends ColorPasswordPipe {
+    transform(password: string) {
+        const passwordArray = Array.from(password);
+        let colorizedPasswordCount = '';
+        for (let i = 0; i < passwordArray.length; i++) {
+            const [character, type] = this.getCharacterAndType(
+                passwordArray[i]
+            );
+            colorizedPasswordCount +=
+                `<span class="password-character password-${type}
+                        ${i % 2 === 0 ? ' bg-grey' : ''}">
```

Comment:
On the `.bg-grey` class application here, I would prefer that we actually create a new class definition for the alternating items, OR, simply use pseudo classes on the stylesheet side to identify even/odd items in the list for coloration so they're "themeable" vs. hard-coding the color class here.